OpenRA.Mods.Common/UtilityCommands/CheckYaml.cs

```diff
@@ -63,29 +77,21 @@ public void Run(ModData modData, string[] args)
 
 				foreach (var testMap in maps)
 				{
-					if (testMap != null)
-					{
-						Console.WriteLine("Testing map: {0}".F(testMap.Title));
-						testMap.PreloadRules();
-					}
-					else
-					{
-						Console.WriteLine("Testing mod: {0}".F(Game.ModData.Manifest.Mod.Title));
-					}
+					Console.WriteLine("Testing map: {0}".F(testMap.Title));
+					testMap.PreloadRules();
+					if (testMap.RuleDefinitions.Any() || testMap.VoiceDefinitions.Any() || testMap.WeaponDefinitions.Any())
```

Comment:
This probably wants to account also for `SequenceDefinitions`, `VoxelSequenceDefinitions`, `NotificationDefinitions` and `SmudgeDefinitions`?